pagopa/pn-frontend
# Feature/PN-10853 - Add FilterStatistics component to sender dashboard

## Short description
Adds a component which let the user filter the statistics in a specific date range

## List of changes proposed in this pull request
- add FilterStatistics component (PA)
- add types definition used to model the new data into model/Statistics.ts
- add a reducer into statisticsSlice to store filter data
- add helper constants to common date.utility.ts
- add it translations for the FilterStatistics component
- remove the 'download jpeg' button from the sender dashboard

## How to test
Prerequisite: As for any feature about the statistics section, a IS_STATISTICS_ENABLED env var should be defined and set to true in order to make visible and reachable the entire section
- Navigate to the statistics section and verify both the instances of the filter properly update applying any change to one of them and statistics charts do the same.
